src/api/wallet/composeProvider.ts

```diff
@@ -12,6 +12,15 @@ import { isWalletConnectProvider, Provider } from './providerUtils'
 import { logDebug } from 'utils'
 import { web3 } from 'api'
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const sanitizeErrorData = (jsonRpcError?: JsonRpcError<any>): void => {
+  if (!jsonRpcError) return
```

Comment:
can jsonRpcError be an empty object?

Reply:
It shouldn't, as long as types are respected, and just in general.
But even if it is, it won't break